github-git/0-first-contribution.md

```diff
@@ -0,0 +1,66 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+(first-contribution)=
+# Make Your First Open Source Contribution 
+
+:::{admonition} What you will learn
+
+After completing this tutorial, you will have the skills needed to contribute to an open source GitHub 
+repository. You will use the [pyOpenSci example GitHub repository](https://github.com/pyOpenSci/pyos-demo-package-contribute) to practice making contributions. 
+:::
+
+Contributing to open source in a public space like GitHub can feel intimidating. You may not know the project maintainers, feel unsure about your GitHub skills, or wonder where to begin.
+
+This lesson series will teach you how to collaborate on code and documentation using GitHub. It will help you confidently contribute to a GitHub repository and build skills to collaborate with colleagues (some of whom you may not have met in real life!). 
+
+## GitHub workflow summary
+
+
+:::{figure} /images/github/steps-to-contribute.png
+:alt: alt text here
+
+The steps for making your first contribution to an open source repo. In this lesson, you will do all of your work on GitHub.com, meaning you don't need to clone or make a copy of your repository locally. This is a great way to get started with contributing. It minimizes the command line skills that you will need to use git. 
+:::
+
+## Documentation counts! 
```

Comment:
This comes a little bit out of nowhere, we are still in overview mode and we're already talking about what kind of PRs we can make. Seems like this would fit better in "identify issue"

Reply:
It does! I'm going to address these comments now and then update this doc, as I did rework this section a bit. But now i see Jeremiah's comment below about small pr's not being welcome in some projects! i just didn't know that was a thing.